src/tests/ftest/soak/soak_test_base.py

```diff
@@ -603,10 +611,30 @@
             if self.loop == 1 and run_harasser:
                 self.harasser_loop_time = loop_time
             self.loop += 1
-        # TO-DO: use IOR
-        if not resv_cont.read_objects():
+        # verify reservered container data
+        final_resv_file = os.path.join(
+            os.environ["DAOS_TEST_LOG_DIR"], "final", "resv_file")
+        try:
+            reserved_file_copy(self, final_resv_file, self.pool[0], resv_cont)
+        except CommandFailure as error:
+            raise SoakTestError(
```

Comment:
(pylint-raise-missing-from) Consider explicitly re-raising using the 'from' keyword